File: C++ Programming/OOP.cpp
#include<iostream>



using namespace std;

class demo 
{
    public:
        int no1;
        int no2;

        void fun()
        {
            cout<<"Inside fun \n";
        }
};


int main()
{
    demo obj1;
    demo obj2;

    cout<<"the size of object is "<<sizeof(obj1)<<"\n";

    obj1.no1 = 10;
    obj1.no2 = 20;

    obj2.no1 = 30;
    obj2.no2 = 40;

    obj1.fun();

    cout<<obj1.no2<<"\n";

    return 0;
}

File: Java Programming/OOP.java
class OOP
{
    public static void main(String Arg[])
    {
        System.out.println("Inside main function");

        Arithmetic obj1 = new Arithmetic();
        Arithmetic obj2 = new Arithmetic(11 , 10);

        int iret  = 0;
        iret = obj1.Addition();
        System.out.println("Addition is :" + iret);
        iret = obj1.Subtraction();
        System.out.println("Subtraction is :" + iret);
        iret = obj2.Addition();
        System.out.println("Addition is :" + iret);
        iret = obj2.Subtraction();
        System.out.println("Subtraction is :" + iret);
    }   
}

class Arithmetic
{
    public int no1;
    public int no2;

    public Arithmetic() //Default constructor
    {
        System.out.println("Inside Default Constructor");
        this.no1 = 0;
        this.no2 = 0;
    }

    public Arithmetic(int i , int j)//Parametrised constructor
    {
        System.out.println("Inside Parametrised Constructor");
        this.no1 = i;
        this.no2 = j;
    }

    public int Addition()
    {
        int Ans = 0;
        Ans = this.no1 + this.no2;
        return Ans;
    }

    public int Subtraction()
    {
        int Ans = 0;
        Ans = this.no1 - this.no2;
        return Ans;
    }
} 